src/ui/Table/TableResponsive.stories.tsx

```diff
@@ -0,0 +1,152 @@
+import type { Meta, StoryObj } from '@storybook/react'
+import {
+  createColumnHelper,
+  flexRender,
+  getCoreRowModel,
+  getSortedRowModel,
+  useReactTable,
+} from '@tanstack/react-table'
+import cs from 'classnames'
+
+import './Table.css'
+
+interface Person {
+  name: string
+  title: string
+  email: string
+  githubHandle: string
+}
+
+const people: Person[] = [
+  {
+    name: 'Lindsay Walton',
+    title: 'Front-end Developer',
+    email: 'lindsay.walton@example.com',
+    githubHandle: 'lindsay203',
+  },
+  {
+    name: 'McGregory James',
+    title: 'Back-end Developer',
+    email: 'mcgregory@example.com',
+    githubHandle: 'greggreg',
+  },
+  {
+    name: 'Brian Davidson',
+    title: 'Front-end Developer',
+    email: 'brian.davidson@example.com',
+    githubHandle: 'yoloswagens',
+  },
+]
+
+const columnHelper = createColumnHelper<Person>()
+
+/*
+  This responsive table example renders different columns/row data depending on the available screen space.
+
+  className hidden and @lg/table:table-cell are used to hide/show columns at different breakpoints.
+*/
+function ResponsiveExampleWithReactTable() {
+  const columns = [
+    columnHelper.accessor('name', {
+      header: () => 'Name',
+      cell: (info) => (
+        <>
+          {info.renderValue()}
+          {/* Only show Title/Email data when rows are collapsed */}
+          <dl className="font-normal @lg/table:hidden">
+            <dt className="sr-only">Title</dt>
+            <dd className="mt-1 truncate text-gray-700">
+              {info.row.getValue('title')}
+            </dd>
+            <dt className="sr-only @md/table:hidden">Email</dt>
+            <dd className="mt-1 truncate @md/table:hidden">
+              {info.row.getValue('email')}
+            </dd>
+          </dl>
+        </>
+      ),
+    }),
+    columnHelper.accessor('title', {
+      header: () => 'Title',
+      cell: (info) => info.renderValue(),
+    }),
+    columnHelper.accessor('email', {
+      header: () => 'Email',
+      cell: (info) => info.renderValue(),
+    }),
+    columnHelper.accessor('githubHandle', {
+      header: () => 'Github Handle',
+      cell: (info) => '@' + info.renderValue(),
+    }),
+  ]
+
+  const table = useReactTable({
+    data: people,
+    columns,
+    getCoreRowModel: getCoreRowModel(),
+    getSortedRowModel: getSortedRowModel(),
+  })
+
+  return (
+    <div className="tableui">
+      <table>
+        <caption>An responsive example using container queries.</caption>
+        <thead>
+          {table.getHeaderGroups().map((headerGroup) => (
+            <tr key={headerGroup.id}>
+              {headerGroup.headers.map((header) => (
+                <th
+                  key={header.id}
+                  colSpan={header.colSpan}
+                  scope="col"
+                  className={cs({
+                    'hidden @lg/table:table-cell':
+                      header.id === 'title' || header.id === 'email',
+                  })}
```

Comment:
So just noticed this playing around with the table resizing the browser window.

![Screenshot 2023-09-07 at 9 47 36 AM](https://github.com/codecov/gazebo/assets/105234307/cce22173-4c3d-441d-9bf6-73ed72750cba)

I think this may be because above in `columns` the `email` id is being hidden on `@md`, but down here it's being checked only on `@lg`.

Reply:
That was it, fixed now!